include/adp536x.h

```diff
@@ -46,6 +46,9 @@
 #define ADP536X_OC_CHG_THRESHOLD_300mA	0x06
 #define ADP536X_OC_CHG_THRESHOLD_400mA	0x07
 
+/* Macro for setting buck output voltage */
+#define ADP536X_VOUT_BUCK(volt) ((u8_t)(((volt)-0.6)/0.05))
```

Comment:
This macro didn't quite work out in my testing. How about something like this: `((uint8_t)((((volt) * 100 - 60) * 2) / 10))`